### PR TITLE
Add opts and mopts support

### PR DESCRIPTION
**Pull request by davidcaste**
I propose to add two new fields to `lvname` dictionary:
- `opts`, for passing extra options to `lvcreate`. It can be used e.g. to create stripped LVM volumes with the option `-i num_stripes`.
- `mopts`, for establishing options to the `mount` command. It can be used e.g. to establish options such `noatime`, `noexec`, etc.

### CODE REVIEW

**Comment by mrlesmithjr** [comment]
@davidcaste Thanks for this. Can you share an example of the variables defined on how this would look?

**Comment by davidcaste** [comment]
Sorry for the late reply, I really missed your message until now.

For example, image I'm using your role currently to provision an instance which will be the primary member of a MongoDB replicaset.

In order to improve the throughput of the EBS volumes, I want to create a striped LVM with two stripes and a stripe size of 256 kB. If I were calling `lvcreate` directly, the command would be `lvcreate -i2 -I256 -l100%FREE -n data mongodb`. The new field `opts` can be used to pass the parameters `-i2 -I256` to `lvcreate`.

When dealing with MongoDB data volumes, it's recommended to mount the volumes with the options `noatime,noexec`. It can improve significantly the performance and the security. The new field `mopts` can be used to mount a partition with these (or other) options.

In summary, this is an snippet of the variables described in the previous example:

```
lvm_groups:
  - vgname: mongodb
    disks:
      - /dev/xvdb
      - /dev/xvdc
    create: true
    lvnames:
      - lvname: data
        size: 100%FREE
        opts: -i2 -I256
        create: true
        filesystem: xfs
        mount: true
        mntp: /var/lib/mongodb
        mopts: noatime,noexec
manage_lvm: true
```

If you have any other questions, or the example has not been clear enough, please let me know.

Thanks!!

**Comment by mrlesmithjr** [comment]
@davidcaste Sorry for the late response on this as well. Is this still something we should merge? I do believe it makes some logical sense for flexibility.

**Comment by mrlesmithjr** [comment]
@davidcaste I have gone ahead and merged this. I had to use this role today and I like the idea you had here so I am merging this.

Thanks again!
